apiserver/params/network.go

```diff
@@ -577,11 +582,12 @@ type AddSubnetsParams struct {
 // SubnetProviderId must be set, but not both. Zones can be empty if
 // they can be discovered
 type AddSubnetParams struct {
-	SubnetTag        string   `json:"subnet-tag,omitempty"`
-	SubnetProviderId string   `json:"subnet-provider-id,omitempty"`
-	SpaceTag         string   `json:"space-tag"`
-	VLANTag          int      `json:"vlan-tag,omitempty"`
-	Zones            []string `json:"zones,omitempty"`
+	SubnetTag         string   `json:"subnet-tag,omitempty"`
+	SubnetProviderId  string   `json:"subnet-provider-id,omitempty"`
```

Comment:
SubnetProviderId vs ProviderNetworkId

Ideally these would be consistent.
ie
SubnetProviderId
NetworkProviderId

I prefer the other way around but what's done is done